src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,36 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_LOGIN_LENGTH = 6;
+    private static final int MIN_PASSWORD_LENGTH = 6;
+    private static final int MIN_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
+    public RegistrationServiceImpl(StorageDao storageDao) {
+    }
+
     @Override
     public User register(User user) {
-        return null;
+        if (user == null
```

Comment:
let's create three separate methods for checkLogin, checkPassword and checkAge

Reply:
Not fixed